pom.xml

```diff
@@ -354,7 +355,7 @@
 					  <excludeFromFailureFile>exclude-pmd.properties</excludeFromFailureFile>
 				  </configuration>
                 </execution>
-              </executions>              
```

Comment:
I don't know why these blanks exist and make differences in this file. I only added one line \<module>value-object\</module>.